src/Static.ts

```diff
@@ -0,0 +1,25 @@
+import {
+    SectionBlock, ContextBlock, PlainTextElement, Option
+} from "@slack/types";
+
+export class Static {
```

Comment:
Static is a keyword in many programming languages including Typescript. We should rename this to be something like StaticPollOperator

Reply:
How about something more neutral, like `Helpers`?